pattern/Conjunction.java

```diff
@@ -123,6 +123,12 @@ public Traversal traversal(List<Identifier.Variable.Name> filter) {
         return traversal;
     }
 
+    public Traversal traversal() {
+        Traversal traversal = new Traversal();
+        variableSet.forEach(variable -> variable.addTo(traversal));
+        return traversal;
+    }
```

Comment:
how is this different from the above method with an empty set of filters? if you want both, then one should call the other and add the filters inside

Reply:
Fair point!